<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>861</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Требуемая надежность(RELY)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calculate">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>230</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Произвести подсчет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>90</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Построить графики</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Размер базы данных(DATA)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сложность продукта(CPLX)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ограничение объема основной 
памяти(STOR)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>331</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ограничение времени выполнения(TIME)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время реакции компьютера(TURN)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Знание приложений (AEXP)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Способности аналитика(ACAP)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Способности программиста(PCAP)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>231</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Знание виртуальной
машины(VEXP)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>291</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Знание языка
программирования(LEXP)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Использование программных 
инструментов(TOOL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>290</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>261</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Использование современных
методов(MODP)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>231</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Требуемые сроки
разработки(SCED)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="RELY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="DATA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="CPLX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="TIME">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="STOR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="VIRT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ACAP">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="AEXP">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="PCAP">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SCED">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="MODP">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="TOOL">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="VEXP">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="LEXP">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="TURN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменчивость виртуальной
машины(VIRT)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во строк кода(LOC)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="LOC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
    <property name="value">
     <number>430000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Атрибуты программного продукта</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Атрибуты компьютера</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Атрибуты персонала</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Атрибуты проекта</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="work">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Трудозатраты без планирования</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>560</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Трудозатраты с планированием</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="work_plan">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>480</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время без планирования</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>550</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время с планированием</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="time_plan">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>560</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="time">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>480</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="normal">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Обычный</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="inter">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Промежуточный</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="inbuilt">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>440</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Встроенный</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>630</y>
      <width>631</width>
      <height>221</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Планирование и определение требований</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Проектирование продукта</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Детальное проектирование</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Кодирование и тестирование отдельных модулей</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Интеграция и тестирование</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Трудозатраты</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>361</height>
     </size>
    </property>
    <row>
     <property name="text">
      <string>Анализ требований</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Проектирование продукта</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Программирование</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Планирование и тестирование</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Верификация и аттестация</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Канцелярия проекта</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Управление конфигурацией</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Создание руководств</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Трудозатраты</string>
     </property>
    </column>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
